ironfish/src/rpc/routes/rpc/getStatus.ts

```diff
@@ -117,6 +119,37 @@ function getRpcStatus(node: IronfishNode): GetRpcStatusResponse {
       formatted.inbound = Math.max(adapter.inboundTraffic.rate1s, 0)
       formatted.outbound = Math.max(adapter.outboundTraffic.rate1s, 0)
       formatted.clients = adapter.clients.size
+    } else if (adapter instanceof RpcHttpAdapter) {
+      formatted.inbound = Math.max(adapter.inboundTraffic.rate1s, 0)
+      formatted.outbound = Math.max(adapter.outboundTraffic.rate1s, 0)
+      formatted.readBytes = adapter.inboundBytes.value
+      formatted.writtenBytes = adapter.outboundBytes.value
+
+      adapter.requests.forEach((r) => {
+        if (!r.req.url) {
+          return
+        }
+        const url = new URL(r.req.url, `http://${r.req.headers.host || 'localhost'}`)
+        const route = url.pathname.substring(1)
```

Comment:
the RpcHttpAdapter adapter also has this logic for parsing the route from the request url. should we pull that out into a method?